clients/gen/common.sh

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-OPENAPI_GENERATOR_CLI_VER=5.4.0
+OPENAPI_GENERATOR_CLI_VER=6.6.0
```

Comment:
We might want to do that in a separate PR which this one would depend on.

Reply:
What’s the benefit of doing that?

Reply:
In a separate PR, you mean? The generator version when upgrading to a new major version contains breaking changes and since the version is shared between all our clients (python and go, too), I think it is worth its own PR.